File: C++ WorldSimulatorGame/POproj1/Owca.cpp
#include "Owca.h"

Owca::Owca(Punkt polozenie, Swiat* swiat)
	:Zwierze(4, 4, 'O', 0, true, Organizmy::owca, polozenie, swiat) {
	this->sila = 4;
	this->inicjatywa = 4;
	this->znak = 'O';
	this->wiek = 0;
}
string Owca::getNazwaOrganizmu() {
	return "Owca";
}

File: Python WorldSimulatorGame/Owca.py
from Zwierze import Zwierze


class Owca(Zwierze):

    def __init__(self, polozenie, swiat):
        super(Owca, self).__init__(4, 4, 'O', 0, True, polozenie, swiat)

    def getNazwaOrganizmu(self):
        return "Owca"